pinot-controller/src/main/resources/app/pages/TenantDetails.tsx

```diff
@@ -384,7 +384,7 @@ const TenantPageDetails = ({ match }: RouteComponentProps<Props>) => {
       setShowReloadStatusModal(true);
       const [reloadStatusData, tableJobsData] = await Promise.all([
         PinotMethodUtils.reloadStatusOp(tableName, tableType),
-        PinotMethodUtils.fetchTableJobs(tableName),
+        PinotMethodUtils.fetchTableJobs(tableName, "RELOAD_SEGMENT,RELOAD_ALL_SEGMENTS"),
```

Comment:
Should we include `FORCE_COMMIT` here?

Reply:
This is the controller screen that shows the reload jobs. This change was needed otherwise it also lists the new job type. @jayeshchoudhary for reviewing the frontend part.

Reply:
Actually, there is a better way to do this. 
We can declare a type with all the reload job types and pass array of types in the function 

And when making api call do 
```js
let searchParams = new URLSearchParams();
if(jobTypes) {
   jobTypes.forEach((jobType) => {
        searchParams.append('jobTypes',  jobType);
   })
}

axios.get('http://server/status', {params: searchParams});
```